specification/components/examples/ScrUploadBundle.json

```diff
@@ -0,0 +1,559 @@
+{
+  "resourceType": "Bundle",
+  "id": "f98f991d-d52c-4b62-9668-3dcc6c27fc50",
+  "identifier": {
+    "system": "https://tools.ietf.org/html/rfc4122",
+    "value": "003D8B36-12D9-11EB-9062-005056276454"
+  },
+  "type": "document",
+  "timestamp": "2020-11-17T01:00:00Z",
+  "entry":  [
+    {
+      "fullUrl": "urn:uuid:2880eedf-76f0-4f38-9681-dc158e937da7",
+      "resource": {
+        "resourceType": "Composition",
+        "id": "2880eedf-76f0-4f38-9681-dc158e937da7",
+        "identifier": {
+          "system": "https://tools.ietf.org/html/rfc4122",
+          "value": "003D8B36-12D9-11EB-9062-005056276454"
```

Comment:
What's the difference between `identifier.value` and `resource.id`? Why we have 2 ids?